src/backend/electron.js

```diff
@@ -0,0 +1,27 @@
+const { app, BrowserWindow } = require('electron');
+
+require('./index'); // main express app
+
+let win;
+
+function createWindow() {
+    mainWindow = new BrowserWindow({
+        width: 1280,
+        height: 720
+    });
+    mainWindow.loadURL('http://localhost:8000/');
```

Comment:
This should be dynamic, because it won't work if parameters are redefined:
* The host should use `--bindaddress`
* The port should use `--port`